setup.cfg

```diff
@@ -20,6 +20,7 @@ classifiers =
     Programming Language :: Python :: 3.9
     Programming Language :: Python :: 3.10
     Programming Language :: Python :: 3.11
+    Programming Language :: Python :: 3.12
```

Comment:
By declaring this. we're committing to fixing any Python 3.12 compatibility issues before each Briefcase release from now on, which should be manageable. But by adding it to CI the same way as the other versions, we would be forced to fix any Python 3.12 issues *immediately*, which could be inconvenient, and isn't necessary so early in the 3.12 release cycle.

Can we use the `continue-on-error` setting as shown [here](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-preventing-a-specific-failing-matrix-job-from-failing-a-workflow-run) to make Python 3.12 failures non-fatal for now? Then we can make them fatal next year when it reaches RC status.

Reply:
Good call. Fixed.